sjb/actions/script.py

```diff
@@ -30,11 +30,13 @@ class ScriptAction(Action):
     the repository as the working directory.
     """
 
-    def __init__(self, repository, script, title, timeout=7200):
+    def __init__(self, repository, script, title, timeout):
         self.repository = repository
         self.script = script
         if title == None:
             title = _SCRIPT_TITLE
+        if timeout == None:
```

Comment:
no i'm not because the caller is explicitly setting the arg to None(by default), which means "timeout=14400" has no effect.